src/transformations/execute.ts

```diff
@@ -232,6 +233,45 @@ export async function executeTransformations(
         vimState.editor.selection = new vscode.Selection(newPos, newPos);
         break;
 
+      case 'executeNormal':
+        const keystroke = keystrokesExpressionParser.parse(transformation.keystroke);
```

Comment:
may not support `:normal A"`

Reply:
thanks.
I created another parser to support `"`.
[255b9f4](https://github.com/VSCodeVim/Vim/pull/8896/commits/255b9f458bfc643691e0c2a1bac8952aeac8fb4c)